.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Epic (Internal Use)
+about: This issue is used to track a feature/project that may span several days
+  to weeks to implement. This is used by the core Metabase team to project plan
+  and manage work.
+title: "[Epic] Title"
+labels: ".Epic"
+
+---
+
+**Links**
+- product doc: _link to product doc_
+- eng doc: _link to technical design doc_
```

Comment:
Can probably call this out as optional, not all projects require a technical design doc